hazelcast/src/main/java/com/hazelcast/map/impl/LocalMapStatsProvider.java

```diff
@@ -176,7 +180,13 @@ private void addStatsOfNoDataIncludedMaps(Map statsPerMap) {
             String mapName = mapProxy.getName();
 
             if (mapConfig.isStatisticsEnabled() && !statsPerMap.containsKey(mapName)) {
-                statsPerMap.put(mapName, EMPTY_LOCAL_MAP_STATS);
+                // Lite members can invoke MapOperations, and their statistics are of importance for monitoring
+                // when Lite members are in use - so we should include proxy stats if they are non-zero
+                if (mapProxy.getLocalMapStats().total() > 0) {
+                    statsPerMap.put(mapName, mapProxy.getLocalMapStats());
+                } else {
+                    statsPerMap.put(mapName, EMPTY_LOCAL_MAP_STATS);
+                }
```

Comment:
to simplify further, maybe we can only have this: 
```suggestion
                    statsPerMap.put(mapName, mapProxy.getLocalMapStats());
```

Advantage is; there can be some other local stats which we can't see if we put `mapProxy.getLocalMapStats().total()` can check as a pre-condition, like map-creation-time or may some near-cache specific ones. By doing this we will not need EMPTY_LOCAL_MAP_STATS and can remove it.

Reply:
Thanks for the feedback @ahmetmircik, and good point regarding using a `.total()` pre-condition - I've pushed another commit which drops this pre-condition and instead introduces a check to see if there are any `LocalMapStatsImpl` already existing to pass. Since `MapProxySupport#getLocalMapStats` is a creative method, we could be creating new objects that serve no purpose; under the hood it also calls back to the `LocalMapStatsProvider` we're already in anyway, so I figured we can cut out the middle man here and access directly (without being creative).

I also realised during testing that it's possible for a Lite member to have a `LocalMapStatsImpl` present without a `MapContainer` (even prior to my changes), so I added some code to `MapServiceContextImpl#destroyMap` to handle this circumstance.

Commit with more details in its description: https://github.com/hazelcast/hazelcast/pull/24871/commits/cdde6f0581aecaf9fe05caec1d5f6b2fcce358f5